scripts/gen_release_requirements.py

```
@@ -0,0 +1,153 @@
+# 用户端 pip 只按本清单精确安装，不再拉取 QtWebEngine。
+# 「日常任务帮助」内嵌 WebView 在无 QtWebEngine 时会自动回退到系统浏览器
+# （src/tasks/daily/daily_liaison_mixin.py open_help_link）。
+# 重新生成：
+#   python scripts/gen_release_requirements.py
+# 校验（CI）：
+#   python scripts/gen_release_requirements.py --check
+"""
+
+
+def generate_compiled() -> str:
+    """从 uv.lock 导出 requirements 格式清单（与锁文件版本一致，不含 dev 组）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)